lib/ae_mdw_web/router.ex

```diff
@@ -52,24 +52,19 @@ defmodule AeMdwWeb.Router do
     get "/txs/:direction", TxController, :txs
     get "/txs/:scope_type/:range", TxController, :txs
 
-    get "/name/auction/:id", NameController, :auction
-    get "/name/pointers/:id", NameController, :pointers
-    get "/name/pointees/:id", NameController, :pointees
-    get "/name/:id", NameController, :name
-
     get "/names/search/:prefix", NameController, :search
-    get "/names/owned_by/:id", NameController, :owned_by
-
     get "/names/auctions", NameController, :auctions
     get "/names/auctions/:scope_type/:range", NameController, :auctions
-
     get "/names/inactive", NameController, :inactive_names
     get "/names/inactive/:scope_type/:range", NameController, :inactive_names
-
     get "/names/active", NameController, :active_names
     get "/names/active/:scope_type/:range", NameController, :active_names
-
     get "/names", NameController, :names
+    get "/names/:id/auction", NameController, :auction
+    get "/names/:id/pointers", NameController, :pointers
+    get "/names/:id/pointees", NameController, :pointees
+    get "/names/:id", NameController, :name
+    get "/names/:id/owned_by", NameController, :owned_by
```

Comment:
Only for this path it would be good to keep the `get "/names/owned_by/:id"` because differently from the others the `:id` is not a name but an account.